src/OpenCensus.Collector.AspNetCore/OpenCensus.Collector.AspNetCore.csproj

```diff
@@ -17,10 +17,6 @@
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
should the same be done for dependencies project?

Reply:
If you mean PackageRequireLicenseAcceptance flag - it is true in both projects.
If you mean creating a separate subfolder and moving away from common, I kept the common thinking it may be useful if we have another listener based on HttpRequestMessage. We could still reuse code across listeners, just could not accommodate AspNetCore due to the different return type. Let me know if we want to eliminate common/ and just dup the code per listener type.

Reply:
No I meant move files from `Common` folder to specific folder under the project folder

Reply:
Moved.